examples/bookstore/bookstore.proto

```diff
@@ -36,34 +36,41 @@ message Error {
   string message = 2;
 }
 
+//CreateShelfParameters holds parameters to CreateShelf
```

Comment:
I believe Google style is to use a space after // in comments.

Reply:
These comments are auto-generated by gnostic-grpc and are constructed in the [surface model](https://github.com/google/gnostic/blob/0320d74b3646387ccc2b21d8bf7a547976170767/surface/model_openapiv3.go#L172) in gnostic.

I don't think you have to worry about these comments right now, Raul.

Reply:
Ah that makes sense. Thanks for providing this info!